packages/x-tailwindcss/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
+  "rootDir": "src",
```

Comment:
why do we need the `rootDir`? What's the difference here? at least I don't see any testing it locally.

Reply:
As we are only including in this tsconfig files from `src` it is unnecesary (moreover it should be under `compilerOptions`, there it is doing nothing). I tried to use a single tsconfig instead of 2 but I found it to be more easy to mess up.